pkg/cmd/eventlistener/delete_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eventlistener
+
+import (
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/tektoncd/cli/pkg/test"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	triggertest "github.com/tektoncd/triggers/test"
+	tb "github.com/tektoncd/triggers/test/builder"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rtesting "knative.dev/pkg/reconciler/testing"
+)
+
+func TestEvetListenerDelete(t *testing.T) {
+	ns := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ns",
+			},
+		},
+	}
+
+	seeds := make([]triggertest.Clients, 0)
+	for i := 0; i < 3; i++ {
+		els := []*v1alpha1.EventListener{tb.EventListener("el-1", "ns")}
+		ctx, _ := rtesting.SetupFakeContext(t)
+		cs := triggertest.SeedResources(t, ctx, triggertest.Resources{EventListeners: els, Namespaces: ns})
+		seeds = append(seeds, cs)
+	}
+
+	testParams := []struct {
+		name        string
+		command     []string
+		input       triggertest.Clients
+		inputStream io.Reader
+		wantError   bool
+		want        string
+	}{
+		{
+			name:        "Invalid namespace",
+			command:     []string{"rm", "el-1", "-n", "invalid"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   true,
+			want:        "namespaces \"invalid\" not found",
+		},
+		{
+			name:        "With force delete flag (shorthand)",
+			command:     []string{"rm", "el-1", "-n", "ns", "-f"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   false,
+			want:        "EventListener deleted: el-1\n",
+		},
+		{
+			name:        "With force delete flag",
+			command:     []string{"rm", "el-1", "-n", "ns", "--force"},
+			input:       seeds[1],
+			inputStream: nil,
+			wantError:   false,
+			want:        "EventListener deleted: el-1\n",
+		},
+		{
+			name:        "Without force delete flag, reply no",
+			command:     []string{"rm", "el-1", "-n", "ns"},
+			input:       seeds[2],
+			inputStream: strings.NewReader("n"),
+			wantError:   true,
+			want:        "canceled deleting eventlistener \"el-1\"",
```

Comment:
```suggestion
			want:        "canceled deleting EventListener \"el-1\"",
```

Can we capitalize this to make consistent with rest of command?

Reply:
This is coming from options. If we change here, then we would need to change in other commands also.